src/main/java/org/apache/commons/text/similarity/LevenshteinDistance.java

```diff
@@ -270,12 +270,6 @@ private static int limitedCompare(CharSequence left, CharSequence right, final i
             final int max = j > Integer.MAX_VALUE - threshold ? n : Math.min(
                     n, j + threshold);
 
-            // the stripe may lead off of the table if s and t are of different
-            // sizes
-            if (min > max) {
```

Comment:
Now we know that m - n <= threshold. Therefore,

j - threshold > n cannot be true, because j's maximum value is m, and in that case the inequality is equivalent to:

m - m > threshold

Reply:
I'll try to give this a look in the morning (UTC-4), and many thanks for the contribution.

Reply:
Hi @chtompki,

Did you get a chance to give this a look? The change will probably require a little extra clean up to remove comments about the "stripe running off the matrix" issue.